### PR TITLE
Clarification of SNS topic policy required for CloudWatch Events in cloudformation

### PR DESCRIPTION
**Pull request by ByronPogson**
Updating aws-resource-events-rule to include TopicPolicy to match permissions created for Lambda example

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/aws-resource-events-rule.md [hunk 1]
```diff
@@ -266,6 +266,22 @@ The following example creates a rule that notifies an Amazon Simple Notification
...
+    "PolicyDocument": {
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Principal": { "Service": "events.amazonaws.com" },
+          "Action": "sns:Publish",
+          "Resource": "*"
+        }
+      ],
+      "Topics": [ "MySNSTopic" ]
```
Comment (AWSRandall) [inline doc_source/aws-resource-events-rule.md]: This needs a Ref for "MySNSTopic".
Could you please rewrite and resubmit?

File: doc_source/aws-resource-events-rule.md [hunk 2]
```diff
@@ -288,4 +304,16 @@ OpsEventRule:
...
           Ref: "MySNSTopic"
         Id: "OpsTopic"
+EventTopicPolicy:
+  Type: "AWS::SNS::TopicPolicy"
+  Properties:
+    PolicyDocument:
+      Statement:
+        - Effect: Allow
+          Principal: 
+            Service: "events.amazonaws.com
```
Comment (AWSRandall) [inline doc_source/aws-resource-events-rule.md]: Missing an end quote at the end of events.amazonaws.com
(Or no quotes at all)

**Comment by ByronPogson** [comment]
Thanks for the feedback Randall, updated.

**Comment by ckesser** [comment]
Thanks for resubmitting, @ByronPogson! I'm merging your request now.

**Pull request merged**